torch/_dynamo/convert_frame.py

```diff
@@ -609,6 +607,9 @@ def log_bytecode(prefix, name, filename, line_no, code):
             if hook_output is not None:
                 out_code = hook_output
 
+        orig_code_map[out_code] = code
+        output_codes.add(out_code)
+
```

Comment:
Since the bytecode hook can modify the output code, the `orig_code_map` and `output_codes` should save the `out_code` after the bytecode hook.